cdk.json

```diff
@@ -0,0 +1,70 @@
+{
+  "app": "npx tsx bin/app.ts",
+  "requireApproval": "never",
+  "output": ".cdk/cdk.out",
+  "watch": {
+    "include": ["**"],
+    "exclude": [
+      "README.md",
+      "cdk*.json",
+      "**/*.d.ts",
+      "**/*.js",
+      "tsconfig.json",
+      "package*.json",
+      "node_modules",
+      "test",
+      "bun.lockb"
```

Comment:
we don't use bun so I think we can remove this line

Reply:
Thanks